week-1-practise/level-3/1-piping.js

```diff
@@ -28,4 +30,4 @@ function format(num) {
 }
 
 console.log("Expected result: 20.00");
-console.log("Actual result: " + result);
+console.log("Actual result: " + result());
```

Comment:
It looks like you're trying to call result like it's a function. It should be a string though, right?